store.go

```diff
@@ -39,6 +41,37 @@ func (c *controller) initDataStore() error {
 	return c.watchNetworks()
 }
 
+func (c *controller) initLocalDataStore() error {
+	c.Lock()
+	cfg := c.cfg
+	c.Unlock()
+	// init local datastore
+	if !c.validateLocalDatastoreConfig() {
+		return fmt.Errorf("local datastore initialization requires a valid configuration")
```

Comment:
Today we use `/var/lib/docker/network/files` in endpoint.go.
@mrjana do you think we can use the same location as the botldb default location ?